client/package.json

```diff
@@ -11,7 +13,10 @@
     "axios-mock-adapter": "^1.18.1",
     "cors": "^2.8.5",
     "enzyme-adapter-react-16": "^1.15.2",
+    "loadash": "^1.0.0",
```

Comment:
Could this have been a typo? :) loadash is deprecated and it's recommended to use lodash only.